src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,15 @@
 package core.basesyntax;
 
 public class MainApp {
+    private Machine truck = new Truck();
+    private Machine bulldozer = new Bulldozer();
+    private Machine excavator = new Excavator();
+    private Machine[] machines = {truck, bulldozer, excavator};
 
+    public static void runMachines(Machine[] machines) {
```

Comment:
You need to create public static void main and call there runMachines method.  And do not make runMachines as a static.